Backend/app/rider-platform/rider-app/Main/src/SharedLogic/MessageBuilder.hs

```diff
@@ -46,3 +48,19 @@ buildSendOTPMessage merchantId req = do
     merchantMessage.message
       & T.replace (templateText "otp") req.otp
       & T.replace (templateText "hash") req.hash
+
+data BuildSendDashBoardOTPMessageReq = BuildSendDashBoardOTPMessageReq
+  { otp :: Text,
+    amount :: Text
+  }
+  deriving (Generic)
+
+buildSendDashBoardOTPMessage :: (EsqDBFlow m r, CacheFlow m r) => Id DM.Merchant -> BuildSendDashBoardOTPMessageReq -> m Text
+buildSendDashBoardOTPMessage merchantId req = do
+  merchantMessage <-
+    QMM.findByMerchantIdAndMessageKey merchantId DMM.SEND_DASHBOARD_OTP
```

Comment:
run in replica

Reply:
This is a cached query. Not able to do runInReplica (Constraint problems are arising)